### DISCUSSION TITLE
JSBsim turboprops: TOT/EGT ?

### DISCUSSION THREAD

**Discussion by magnetic-eagle** [JSBSim-Team/jsbsim#584] [Support requests]
Hello !
I'm currently implementing the turboprop engine of a FlightGear aircraft, which instead of an ITT gauge has a TOT gauge ! But sadly I couldn't find a TOT/EGT property anywhere. Do you think it would be possible to add this to JSBsim, or could I at least calculate it from some other properties using a system function ?  
Should not be that hard as it's already implemented for turbine engines, but I couldn't hack anything together by looking at the source code.

Using an ITT gauge instead of an TOT gauge is sadly not possible because there are no references for that in the POH of the aircraft, only TOT values are given.

**Comment by dpculp** [comment]
In FlightGear you can use scale and factor tags to modify an existing 
property.  Shouldn't be too hard to use these to make ITT look like the 
EGT you expect from the POH.

-- Dave

**Reply by magnetic-eagle** [reply]
Luckily another aircraft has the exact same engine with a working TOT gauge, so I can hack something up. They are calculating the TOT from `atmosphere/rho-slugs_ft3` and the torque,
